samples/net/sockets/big_http_download/src/big_http_download.c

```diff
@@ -144,6 +149,19 @@ void download(struct addrinfo *ai, bool is_tls)
 	CHECK(sock);
 	printf("sock = %d\n", sock);
 
+#if defined(CONFIG_NET_SOCKETS_SOCKOPT_TLS)
+	if (is_tls) {
+		sec_tag_t sec_tag_opt[] = {
+			CA_CERTIFICATE_TAG,
+		};
+		CHECK(setsockopt(sock, SOL_TLS, TLS_SEC_TAG_LIST,
+				 sec_tag_opt, sizeof(sec_tag_opt)));
+
+		CHECK(setsockopt(sock, SOL_TLS, TLS_HOSTNAME,
+				 host, strlen(host) + 1));
```

Comment:
This strlen() + 1 looks a bit weird, I'm sure will be source of common mistake ;-)